### PR TITLE
Change the condition to detect the 404 page.

### LINKED ISSUE #24: HangHub is not loading when open issue/PR from the list.
When you open any repo and issue/PR list and open an issue HangHub doesn't load properly. After refresh, all seems to work fine.

### PR DESCRIPTION
**Pull request by elszczepano**
The previous condition worked also on issues/PRs list. 
Closes #24.

### CODE REVIEW

File: frontend/src/index.js [hunk 1]
```diff
@@ -44,7 +45,7 @@ function canStart( config, hostname ) {
 	}
 
 	// Check if user opened page without required permissions.
-	if ( !document.getElementById( 'partial-discussion-header' ) ) {
+	if ( document.querySelectorAll( 'img.js-plaxify' )[ 1 ] && document.querySelectorAll( 'img.js-plaxify' )[ 1 ].alt.startsWith( '404' ) ) {
```
Comment (ma2ciek) [inline frontend/src/index.js]: What is the `img.js-plaxify` item? You could describe it somehow. And can't we use the first matched element?

Comment (ma2ciek) [reply frontend/src/index.js]: I think also that having only `document.querySelectorAll( 'img.js-plaxify' )[ 1 ].alt.startsWith( '404' )` might quickly break after some minor GH site changes.

Comment (elszczepano) [reply frontend/src/index.js]: `img.js-plaxify` is a part of the 404 page. This page contains an octocat graphic built with some `img` tags. The second element of this array is the only which contains information about 404 error. I'll add a comment with the explanation in the code.

Comment (ma2ciek) [reply frontend/src/index.js]: But are `img.js-plaxify` elements only present to the `Page not found`? If yes then we can just check the `!!document.querySelector( 'img.js-plaxify' )`

Comment (ma2ciek) [reply frontend/src/index.js]: Actually, any error leading to this page should remove the user from the collaboration. 404 is only one of them.

Comment (elszczepano) [reply frontend/src/index.js]: > But are `img.js-plaxify` elements only present to the `Page not found`? If yes then we can just check the `!!document.querySelector( 'img.js-plaxify' )`

That should be enough.

Comment (elszczepano) [reply frontend/src/index.js]: But there's no need to double negation. `document.querySelector( 'img.js-plaxify' )` returns `null` if element doesn't exist.

Comment (ma2ciek) [reply frontend/src/index.js]: yep, the `!!` only converts it to boolean.

**Pull request merged**